aikau/src/main/resources/alfresco/forms/Form.js

```diff
@@ -1037,6 +1050,11 @@ define(["dojo/_base/declare",
                this.publishValidValue();
             }
             this._formSetupComplete = true;
+
+            // If configured, focus the first field in the form
+            if (this.firstFieldFocusOnLoad) {
+               setTimeout(lang.hitch(this, this.focus));
```

Comment:
Didn't work without it. Worked reliably with it. Figure other processes are running on the page. This lets every synchronous process run and then executes the focus.